pom.xml

```diff
@@ -27,6 +38,36 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+
```

Comment:
add all common dependencies from others pom here (as spring data jpa, starter-test) and delete unused